addon/authenticators/oauth2-password-grant.js

```diff
@@ -141,7 +141,10 @@ export default BaseAuthenticator.extend({
     @deprecated OAuth2PasswordGrantAuthenticator/rejectWithResponse:property
     @public
   */
-  rejectWithXhr: computed.deprecatingAlias('rejectWithResponse'),
+  rejectWithXhr: computed.deprecatingAlias('rejectWithResponse', {
+    id: 'esa-deprecate.reject-with-xhr',
```

Comment:
We're using deprecation ids following the pattern `ember-simple-auth.<topic>.<id>` generally, see e.g. https://github.com/simplabs/ember-simple-auth/blob/726e59bb3c966a2f5cab5075a510a165f8c7a46c/addon/configuration.js#L91